### PR TITLE
Brings basic ld+json structured dadta

### PR DESCRIPTION
**Pull request by softr8**
## Quick Info
Using LD+JSON format to inject structured data to crawlers,
it follows rules by schema.org

## What does this change?
Adds structured data to markup

## Why are you changing that?
Crawlers such as google bot can interpret our site better 

## How do you manually test this? 
use google structured data testing tool to read the report

## Screenshots
N/A

### CODE REVIEW

**Review by fedgut — APPROVED** [review]
LGTM!

**Review by kevinnio — APPROVED** [review]
Awesome!

File: app/serializers/schema_blogpost_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        '@type': 'BlogPosting',
+        'headline': entry.title,
+        'alternativeHeadline': 'MagmaLabs Today I Learned',
+        'datePublished': entry.created_at.strftime('%Y-%m-%d'),
+        'dateModified': entry.updated_at.strftime('%Y-%m-%d'),
+        'articleBody': entry.body,
+        'image': context.image_url('logo'),
+        'inLanguage': 'en-US',
+        'copyrightYear': entry.created_at.year,
+        'keywords': %w{#{entry.channel_name} MagmaLabs TIL BlogPost},
```
Comment (EduardoGHdez) [inline app/serializers/schema_blogpost_serializer.rb:27]: > Keywords or tags used to describe this content. Multiple entries in a keywords list are typically delimited by commas.

Hey, @softr8 . Do you know if does it affect not using commas here? 🤔 
Not sure tbh, but would be nice to know

Comment (softr8) [reply app/serializers/schema_blogpost_serializer.rb:27]: I found this to be ambiguous everywhere, but most of them were using an array

**Review by EduardoGHdez — COMMENTED** [review]
source (1)

**Pull request merged**